src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/PdoSessionHandlerTest.php

```diff
@@ -324,15 +324,16 @@ public function testGetConnectionConnectsIfNeeded()
     public function testUrlDsn($url, $expectedDsn, $expectedUser = null, $expectedPassword = null)
     {
         $storage = new PdoSessionHandler($url);
+        $reflection = new \ReflectionClass(PdoSessionHandler::class);
 
-        $this->assertAttributeEquals($expectedDsn, 'dsn', $storage);
+        foreach (['dsn' => $expectedDsn, 'username' => $expectedUser, 'password' => $expectedPassword] as $property => $expectedValue) {
+            if (!isset($expectedValue)) {
+                continue;
+            }
+            $property = $reflection->getProperty($property);
+            $property->setAccessible(true);
```

Comment:
I don't see how to keep the Test without reflection. (unless refactoring the way DSN is parsed, bu it's not necessary IMHO)